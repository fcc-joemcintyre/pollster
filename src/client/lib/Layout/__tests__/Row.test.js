/* eslint-env jest */
import React from 'react';
import renderer from 'react-test-renderer';
import { Row } from '..';

test ('Row renders default layout', () => {
  const component = renderer.create (
    <Row>
      text
    </Row>
  );
  const tree = component.toJSON ();
  expect (tree).toMatchSnapshot ();
});

test ('Row renders left justified content', () => {
  const component = renderer.create (
    <Row left>
      text
    </Row>
  );
  const tree = component.toJSON ();
  expect (tree).toMatchSnapshot ();
});

test ('Row renders centered content', () => {
  const component = renderer.create (
    <Row center>
      text
    </Row>
  );
  const tree = component.toJSON ();
  expect (tree).toMatchSnapshot ();
});

test ('Row renders right justfied content', () => {
  const component = renderer.create (
    <Row right>
      text
    </Row>
  );
  const tree = component.toJSON ();
  expect (tree).toMatchSnapshot ();
});
