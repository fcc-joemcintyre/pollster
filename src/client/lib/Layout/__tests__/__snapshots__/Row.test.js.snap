// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row renders centered content 1`] = `
.c0 {
  text-align: center;
}

<div
  className="c0"
>
  text
</div>
`;

exports[`Row renders default layout 1`] = `
.c0 {
  text-align: left;
}

<div
  className="c0"
>
  text
</div>
`;

exports[`Row renders left justified content 1`] = `
.c0 {
  text-align: left;
}

<div
  className="c0"
>
  text
</div>
`;

exports[`Row renders right justfied content 1`] = `
.c0 {
  text-align: right;
}

<div
  className="c0"
>
  text
</div>
`;
